tests/Tests/ORM/Tools/SchemaValidatorTest.php

```diff
@@ -228,6 +232,51 @@ public function testInvalidAssociationTowardsMappedSuperclass(): void
             $ce,
         );
     }
+
+    public function testBigintMappedToStringInt(): void
+    {
+        $class = $this->em->getClassMetadata(BigintMappedToStringInt::class);
+        $ce    = $this->validator->validateClass($class);
+
+        if (method_exists(BigIntType::class, 'getName')) { // DBAL 3
+            $this->assertEquals([], $ce);
+        } else { // DBAL 4+
+            $this->assertEquals([], $ce);
+        }
```

Comment:
… why are you using a condition here? It's the same code.

Reply:
Well. too much copy-pasted... ;-)